src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss

```diff
@@ -37,5 +46,4 @@
       }
     }
   }
-}
-
+}
```

Comment:
Out linter complains if there is not a new line at the end of each file.

Reply:
same as the other